pkg/sink/codec/common/config.go

```diff
@@ -138,6 +141,8 @@ type urlConfig struct {
 	AvroSchemaRegistry       string `form:"schema-registry"`
 	OnlyOutputUpdatedColumns *bool  `form:"only-output-updated-columns"`
 	ContentCompatible        *bool  `form:"content-compatible"`
+
+	DebeziumDisableSchema *bool `form:"debezium-disable-schema"`
```

Comment:
please add this as a  part of changefeed toml configuration,  not just a sink uri parameter

Reply:
I added to replica config, PTAL again, thanks!